src/main/webapp/js/stats-page-loader.js

```diff
@@ -1,27 +1,27 @@
 // Fetch stats and display them in the page.
-  function fetchStats() {
-    const url = '/stats';
-    fetch(url).then((response) => {
-      return response.json();
-    }).then((stats) => {
-      const statsContainer = document.getElementById('stats-container');
-      statsContainer.innerHTML = '';
+function fetchStats() {
+  const url = '/stats';
+  fetch(url).then((response) => {
+    return response.json();
+  }).then((stats) => {
+    const statsContainer = document.getElementById('stats-container');
+    statsContainer.innerHTML = '';
 
-      const messageCountElement = buildStatElement('Message count: ' + stats.messageCount);
-      statsContainer.appendChild(messageCountElement);
+    const messageCountElement = buildStatElement('Message count: ' + stats.messageCount);
```

Comment:
nit: Not sure if you know about this, but if you don't, then its something new to learn (:

```suggestion
    const messageCountElement = buildStatElement(`Message count: ${stats.messageCount}`);
```

Reply:
Can you please explain the difference between the above two representations? I am not sure I follow..

Reply:
Let
```
const s = 'world';
```
Then
```
`Hello ${s}`
```
results in the exact same string as
```
'Hello ' + s
```
This is called template literals: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals.

Reply:
Ohh ok, thank you!